CHANGELOG.md

```diff
@@ -47,6 +47,7 @@
 - Upgrade reference tests to version 13.1 [#6574](https://github.com/hyperledger/besu/pull/6574)
 - Extend `BesuConfiguration` service [#6584](https://github.com/hyperledger/besu/pull/6584)
 - Add `ethereum_min_gas_price` and `ethereum_min_priority_fee` metrics to track runtime values of `min-gas-price` and `min-priority-fee` [#6587](https://github.com/hyperledger/besu/pull/6587)
+- Extend `Blockchain` service [#6592](https://github.com/hyperledger/besu/pull/6592)
```

Comment:
think this will now be in the next release

Reply:
moved to next release